fastapi/openapi/docs.py

```diff
@@ -14,6 +14,7 @@ def get_swagger_ui_html(
     swagger_favicon_url: str = "https://fastapi.tiangolo.com/img/favicon.png",
     oauth2_redirect_url: Optional[str] = None,
     init_oauth: Optional[Dict[str, Any]] = None,
+    swagger_ui_parameters: Optional[Dict[str, str]] = None,
```

Comment:
I think the second parameter of this line `Dict[str, str]` can be also an integer so I suggest to change it into  `Dict[str, any] `

Reply:
Good point, changed.

Reply:
Nice!